menderqa_pipeline.go

```diff
@@ -58,11 +68,6 @@ func getBuilds(log *logrus.Entry, conf *config, pr *github.PullRequestEvent) []b
 				}
 			}
 
-			// we need to have the latest integration/master branch in order to use the release_tool.py
-			if err := updateIntegrationRepo(conf); err != nil {
```

Comment:
Question:
Why was this needed? :thinking:

Reply:
It would work _most of the times_ keeping it inside the loop.

But the detail is that the day that you add a repo, you need to pull integration before being able to know if it is a repo that needs watching or not. Previously the list was hardcoded so this was not needed.

One might argue that this is slightly less efficient as we are pulling always when before we only pulled once we knew the repo was one to track. But well... we are not optimizing for runtime here, are we?

Reply:
Good to know, thanks, makes sense.
Nah, no worries about runtime here.